Resources/Prototypes/Floof/Traits/physical.yml

```
@@ -103,3 +103,66 @@
+    traits:
+      - Small # conflicts with PseudoItem changes
+  - !type:CharacterSpeciesRequirement 
+    inverted: true
+    species:
+    - Oni # incompatable due to use of the PlayerToolModifier and BonusMeleeDamage - a special trait will need to be made
+  - !type:CharacterDepartmentRequirement
+      inverted: true
+      departments:
+        - Security # No tiny sec, sad
+  functions:
+  - !type:TraitAddComponent
```

YAMLlint (1.35.1)

[warning] wrong indentation: expected 4 but found 2

(indentation)